README.md

```diff
@@ -51,6 +51,8 @@ instance.
 (builds on every commit) or with `mujx/hakatime:latest-arm` (manual semi-regular updates for `arm/v7` & `arm64`)
 or by building the image yourself with the dedicated Dockerfile (`Dockerfile.arm`).
 
+**WARNING**: This stack relies on `./docker` directory located in this repository, please run it after making `git clone` to evade issues.
```

Comment:
Could you rename `stack` to `setup` to not confuse it with the `stack` build tool.

Reply:
Of course, done :ok_hand: